src/me/ryanhamshire/PopulationDensity/PopulationDensity.java

```diff
@@ -1454,28 +1465,14 @@ static void removeMonstersAround(Location location)
             }
         }
 	}
-	
-	private SimpleEntry<Integer, Integer> processMaterials(String string)
+
+	private Material processMaterials(String string)
 	{
-        String [] elements = string.split(":");
-        if(elements.length < 2)
-        {
-            PopulationDensity.AddLogEntry("Couldn't understand config entry '" + string + "'.  Use format 'id:data'.");
-            return null;
-        }
-        
-        try
-        {
-            int id_output = Integer.parseInt(elements[0]);
-            int data_output = Integer.parseInt(elements[1]);
-            return new SimpleEntry<Integer, Integer>(id_output, data_output);
-        }
-        catch(NumberFormatException e)
-        {
-            PopulationDensity.AddLogEntry("Couldn't understand config entry '" + string + "'.  Use format 'id:data'.");
-        }
-        
-        return null;
+        Material material = Material.getMaterial(string);
+        if (material == null) {
+			AddLogEntry("Error: Couldn't resolve material \""+string+"\". Please update your config.yml.");
```

Comment:
It's already here

Reply:
Was thinking along the lines of being a message after startup in log, but at least the more diligent administrators will see this.